updater/lib/dependabot/updater/operations/create_group_update_pull_request.rb

```diff
@@ -64,6 +64,8 @@ def perform
               service.create_pull_request(T.must(dependency_change), dependency_snapshot.base_commit_sha)
             rescue StandardError => e
               error_handler.handle_job_error(error: e, dependency_group: group)
+            ensure
+              service.record_ecosystem_meta(dependency_snapshot.ecosystem)
             end
```

Comment:
**Review Tip:** Send metrics for `create group operation` after update process (success, or fail)